config/helm/copy/cilium.io_v2_ciliumnetworkpolicy_capz-controller-manager.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ciliumNetworkPolicy.enabled }}
```

Comment:
the directory is called config/helm/copy
is that intended?

Reply:
These are the files that are supposed to be copied to the templates directory.

Reply:
They were under `helm/copy` folder but I changed it. I explained why here https://github.com/giantswarm/cluster-api-provider-azure-app/pull/149/files#r1654432163